cmd/runsim/integrations/github/main.go

```diff
@@ -0,0 +1,178 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/cosmos/tools/lib/runsimaws"
+	"github.com/cosmos/tools/lib/runsimgh"
+)
+
+const (
+	// Security token IDs
+	ssmGhAppTokenId  = "github-sim-app-key"
+	ssmCircleTokenId = "circle-token-sim"
+
+	// Github app parameters
+	startSimCmd      = "Start sim"
```

Comment:
what are the differences and options for running the sim?

Reply:
Right now there are no options from github. You can change all the parameters from the Circleci [context](https://circleci.com/gh/organizations/tendermint/settings#contexts/655e3177-bcc3-40d8-a56d-d92c9d7dfad8). I did it this way so that there's less parsing of text from the github comment, less chances for errors.